src/init/CliRunner.ts

```diff
@@ -23,25 +22,35 @@ export const runCustom = function(
   const { argv } = yargs
     .usage('node ./bin/server.js [args]')
     .options({
-      port: { type: 'number', alias: 'p' },
+      port: { type: 'number', alias: 'p', default: 3000 },
       config: { type: 'string', alias: 'c' },
     })
     .help();
 
-  new Promise<RuntimeConfig>(async(resolve): Promise<void> => {
+  new Promise<string>(async(resolve, reject): Promise<void> => {
```

Comment:
The whole promise wrap thingy is weird here. Can't we just make this entire function `async`?

Reply:
…or if we can't, just call an async function that does it?

Reply:
[Link](https://github.com/solid/community-server/pull/74#discussion_r479967520) to @rubensworks reply when I asked the same thing. But yea, moving this part to a separate async function could make it cleaner.